kimchi_navigation/CMakeLists.txt

```diff
@@ -39,44 +39,27 @@ endif()
 # Include directories
 include_directories(include)
 
-add_library(global_localization SHARED
-  src/global_localization.cpp
-)
-target_include_directories(global_localization PUBLIC
-  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-  $<INSTALL_INTERFACE:include>
-)
-target_compile_definitions(global_localization
-  PRIVATE "CUSTOM_ACTION_CPP_BUILDING_DLL")
-ament_target_dependencies(global_localization
+add_executable(global_localization src/global_localization.cpp)
```

Comment:
Just for now I'm setting this to be an executable as it seem to be solving a bug